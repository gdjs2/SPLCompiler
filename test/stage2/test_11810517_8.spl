struct st {
	int x;
};

struct st f(int z) {
	struct st s;
	if (z == 1) return s;
	else return s.x;
}