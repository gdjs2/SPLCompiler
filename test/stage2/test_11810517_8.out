Error Type 8 at Line 8: functionâ€™s return value type mismatches the declared type